dask/dataframe/core.py

```diff
@@ -2150,7 +2150,6 @@ def sum(
         else:
             return result
 
-    @_numeric_only
     @derived_from(pd.DataFrame)
```

Comment:
I see you removed the decorator. The decorator did two things, raise on `numeric_only=False`, and filter the underlying data on `numeric_only=True`. Are they both irrelevant now?

Reply:
pandas takes care of filtering for ``numeric_only=True``, we technically don't need it here. Is there a reason why we would want to do it ourselves?

Reply:
+1 for offloading to `pandas` if we can